ci/ray_ci/linux_container.py

```diff
@@ -18,9 +18,11 @@ def __init__(
         docker_tag: str,
         volumes: Optional[List[str]] = None,
         envs: Optional[List[str]] = None,
+        mount_tmpfs: bool = False,
```

Comment:
so by default it will not be tmpfs - still overlayfs? 

When do we use tmpfs (mount_tmpfs=True)in this case?

Reply:
yes, tmpfs only on running tests for now.

on other tasks (e.g. running wrappers to build or test or setup environments), it will still be the same behavior as before.

Reply:
oh i guess i was confused that I didn't see `mount_tmpfs` being set to True, is this done somewhere else?

Reply:
oh.. you are right. my bad.